src/can/can.c

```diff
@@ -223,88 +273,138 @@ void handle_tx_interrupt(mob_t* mob) {
     pause_mob(mob);
 }
 
-void handle_auto_tx_interrupt(mob_t* mob) {
-    select_mob(mob->mob_num);
-
-    // TODO: Many of the variables, including IDTAG, dlc, etc
-    // are copied from the incoming remote frame; thus, they must all be
-    // reset and restored to their original values
-    // In particular, the RTR tag and RPLV bit must be reset to their
-    // original values
-    // as in the TX case, if there is no data left, pause the mob
-    // otherwise, load fresh data via tx callback
-
-    // TODO: maybe make this work like TX MObs?
-
-    load_data(mob);
-
-    // clear interrupt flag
-    CANSTMOB &= ~(_BV(TXOK));
-
-    // this should happen all at once
-    if (mob->dlc != 0) resume_mob(mob);
-    else pause_mob(mob);
-
-    // TODO: for some reason, we need to set these AFTER
-    set_id_tag(mob->id_tag);
-    set_id_mask(mob->id_mask);
-    set_ctrl_flags(mob->ctrl);
-}
-
+// Returns contents of CANSTMOB register
 uint8_t mob_status(mob_t* mob) {
     select_mob(mob->mob_num);
     return CANSTMOB;
 }
 
+// Prints error statements
+// Most errors are handled by automatically by CAN
 uint8_t handle_err(mob_t* mob) {
     select_mob(mob->mob_num);
 
     uint8_t err = mob_status(mob) & 0x9f;
 
     if (err != 0) {
         if (err & _BV(DLCW)) {
-            print(ERR_MSG, "Bad DLC");
+            print(ERR_MSG, "Bad DLC\n");
         } else if (err & _BV(BERR)) {
-            print(ERR_MSG, "Bit error");
+            print(ERR_MSG, "Bit error\n");
         } else if (err & _BV(SERR)) {
-            print(ERR_MSG, "Bit stuffing error");
+            print(ERR_MSG, "Bit stuffing error\n");
         } else if (err & _BV(CERR)) {
-            print(ERR_MSG, "CRC mismatch");
+            print(ERR_MSG, "CRC mismatch\n");
         } else if (err & _BV(FERR)) {
-            print(ERR_MSG, "Form error");
+            print(ERR_MSG, "Form error\n");
         } else if (err & _BV(AERR)) {
-            print(ERR_MSG, "No ack");
+            print(ERR_MSG, "No ack\n");
         }
```

Comment:
Don't need to put `\n` here, see how `ERR_MSG` is defined at the top of this file